behave/features/alternative_help.feature

```diff
@@ -7,15 +7,18 @@ Background: Login
 Scenario: Assign alternative help for out of scope user with necessary details
     GIVEN I am on the 'call centre dashboard' page
     WHEN I select to 'Create a case'
-    THEN I complete the users details with 'Test Dummy User' details
-    THEN I navigate the call centre dashboard
-    THEN I go back to the previous case
-    THEN I should see the users previously entered details
-    WHEN I select ‘Create Scope Diagnosis'
+    AND I complete the users details with 'Test Dummy User' details
+    AND I navigate back to the call centre dashboard
+    AND I go back to the previous case
+    AND I see the users previously entered details
+    AND I select ‘Create Scope Diagnosis'
     AND I select the diagnosis <category> and click next <number> times
     | category                                                | number |
     | Crime                                                   | 2      |
-    THEN I get an "OUTOFSCOPE" decision
-    WHEN I select 'Assign Alternative Help'
-    THEN I am taken to the "Alternative help" page for the case located at "/alternative_help/"
+    AND I get an "OUTOFSCOPE" decision
+    #This is the icon in the top RH corner
+    AND I select 'Assign Alternative Help'
```

Comment:
What do you of changing the step to include it's for an icon

Reply:
Do you mean changing the step so it says `I select 'Assign Alternative Help' icon`?

Reply:
@jogilder yep